packages/communication-react/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Wed, 29 Jun 2022 17:31:05 GMT
   - Fix loading spinner size in small containers ([PR #1995](https://github.com/azure/communication-ui-library/pull/1995) by 2684369+JamesBurnside@users.noreply.github.com)
   - Add aria labels and announcer to file sharing components to make file sharing meet accessibility standards ([PR #1960](https://github.com/azure/communication-ui-library/pull/1960) by 97124699+prabhjot-msft@users.noreply.github.com)
   - Update Participant pane to be an overlay on top of message thread so it can be tapped into when opened ([PR #1943](https://github.com/azure/communication-ui-library/pull/1943) by 97124699+prabhjot-msft@users.noreply.github.com)
+  - Increase Z-index of jump to new message button to 2 so the button stays on top of message bubbles ([PR #2046](https://github.com/Azure/communication-ui-library/pull/2046) by carolinecao@microsoft.com)
```

Comment:
no need to mention what the z index value is.
I suggest keeping it simple.
`Fixed a bug where the "New Message" button was getting hidden under messages`

Reply:
I second Anjul's change here. 
Will approve the review once the change is in :)